HISTORY.md

```diff
@@ -1,4 +1,7 @@
 # Version History
+## 1.1.11 / 2021-06-09
```

Comment:
Should be one more line of whitespace between the first and second line.

Reply:
Wow I missed the date as well